src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -216,6 +218,8 @@ def run(self, user_config: dict, sbom_path: str = "", skip_deps: bool = False) -
                 logger.info(str(report))
 
                 db_session.add(analysis)
+
+                logger.info("PURL of the main target software component is '%s'.", main_record.context.component.purl)
```

Comment:
```suggestion
                logger.info("The PURL string for the main target software component in this analysis is '%s'.", main_record.context.component.purl)
```

Also update the documentation example.

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/f66b0be2bf3c2556eed5885315d39cd66b792306 and the documentation is fixed in https://github.com/oracle/macaron/pull/401/commits/f39f371e085e9f521d45a11e7a498ba126ffc1c3